custom/icds_reports/static/js/directives/navigation/navigation.directive.js

```diff
@@ -3,8 +3,8 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function NavigationController($window, $rootScope, $scope, $route, $routeParams, $location, navigationService,
-                              stateLevelAccess, haveAccessToAllLocations, haveAccessToFeatures,
-                              userFullName, userUsername, isMobile) {
+    stateLevelAccess, haveAccessToAllLocations, haveAccessToFeatures,
+    userFullName, userUsername, isMobile, navMenuItems) {
```

Comment:
I find this indentation weird because the function arguments are at the same level as the function body (but I know stickler likes it this way and seems fine)